packages/template-typescript-minimal/app/overrides/components/with-red-border.jsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+
+// OVERRIDE!! This border is now BLUE.....
+
+// Define the HOC function
+const withRedBorder = (WrappedComponent) => {
+    const WithRedBorder = (props) => {
+        return (
+            <div style={{border: '2px solid blue', padding: '10px'}}>
+                <WrappedComponent {...props} />
+            </div>
+        )
+    }
```

Comment:
Here we override to _replace_ the component. But what if we want to _extend_ it?

Previously we were able to import the original component and reuse it in the overrides file. Can we do something similar here now?

Reply:
Update: it looks like the most recent commits have fixed the issue. I was able to import the original HOC as mentioned above.